mws/apis/products.py

```diff
@@ -163,9 +163,89 @@ def get_lowest_priced_offers_for_asin(
         }
         return self.make_request(data)
 
-    # # # TODO add this
-    # def get_my_fees_estimate(self):
-    #     pass
+    def get_my_fees_estimate(
+        self, marketplace_id, id_type, id_value, is_amazon_fulfilled, identifier, listing_price, listing_price_currency_code
+    ):
+
+        """
+        Each argument must be a list, up to a length of 20, with id_value (number of ASINs or SellerSKUs) determining how many requests are made.
+        Options: 
+            A single element list which repeats, or
+            A list of equal length as id_value for a one to one relationship
+
+        Example:
+            marketplace_id = ['A1F83G8C2ARO7P']
+            id_value = ['B082YTKC47', 'B07WGJ9JGP', 'B01LXP5TXI']
+            id_type = ['ASIN']
+            is_amazon_fulfilled = [True, False, True]
+            identifier = ['request1', 'request2', 'request3']
+            listing_price = [9, 16.42, 2.99]
+            listing_price_currency_code = ['GBP']
+
+        Example 2:
+            marketplace_id = ['A1F83G8C2ARO7P', 'ATVPDKIKX0DER', 'A13V1IB3VIYZZH']
+            id_value = ['B082YTKC47', 'B07WGJ9JGP', 'B01-LXP-TXI6']
+            id_type = ['ASIN', 'ASIN', 'SellerSKU']
+            is_amazon_fulfilled = [True]
+            identifier = ['Fee requests']
+            listing_price = [19]
+            listing_price_currency_code = ['GBP', 'USD', 'EUR']
+            
+            
+        Returns Amazon Fees for ASINs and SellerSKUs specified.
+
+        Docs:
+        https://docs.developer.amazonservices.com/en_UK/products/Products_GetMyFeesEstimate.html
+
+        """
+        
+        data = {
+            "Action": "GetMyFeesEstimate",
+        }
```

Comment:
Minor point, an upcoming change (#207) will move the `Action` parameter to the signature for `make_request`. I recommend removing this key from `data`, and then at the end of the method calling:

```python
return self.make_request("GetMyFeesEstimate", data)
```